Code/Tools/AssetProcessor/native/AssetManager/SourceFileRelocator.cpp

```diff
@@ -886,10 +886,15 @@ Please note that only those seed files will get updated that are active for your
         return report;
     }
 
-    AZ::Outcome<RelocationSuccess, MoveFailure> SourceFileRelocator::Move(const AZStd::string& source, const AZStd::string& destination, bool previewOnly, bool allowDependencyBreaking, bool removeEmptyFolders, bool updateReferences, bool excludeMetaDataFiles)
+    AZ::Outcome<RelocationSuccess, MoveFailure> SourceFileRelocator::Move(const AZStd::string& source, const AZStd::string& destination, int flags )
     {
         AZStd::string normalizedSource = source;
         AZStd::string normalizedDestination = destination;
+        bool previewOnly = (flags & PreviewOnlyFlag) ? true : false;
```

Comment:
nit: It's usually good practice to write `(flags & PreviewOnlyFlag) != 0` to make it an explicit bool check rather than an implicit conversion

Reply:
Changed.